MAUI/DataGrid/styling.md

```diff
@@ -325,6 +325,21 @@ The unbound row can be customized by the writing style for [DataGridUnboundRowVi
 
 <img alt="Unbound row styling in .NET MAUI DataGrid" src="Images\styling\maui-datagrid-unbound-row-styling.png" width="404"/>
 
+### Styling show row header
+The row header can be customized by the writing style for DataGridRowHeaderCell Target Type. 
```

Comment:
The row header can be customized by the writing style for `DataGridRowHeaderCell` Target Type